### PR TITLE
chore: update sanitize-html, ws and transitive dependencies

### PR DESCRIPTION
**Pull request by hitesh-parmar**
Solves #1754 

https://github.com/SAP/open-ux-tools/security/dependabot/116
https://github.com/SAP/open-ux-tools/security/dependabot/68
https://github.com/SAP/open-ux-tools/security/dependabot/117

Update to "sanitize-html": "2.12.1" 
"ws": "8.18.0" 
"@types/ws": "8.5.12"

No vulnerabilities after executing `pnpm audit`

Update transitive dependencies by executing `pnpm update`

### CODE REVIEW

**Review by Klaus-Keller — DISMISSED** [review]
Version bump and linting fixes look good. Approved from my side.
Review from @SAP/ux-tools-adp  is required.

**Review by donal-tobin-sap — APPROVED** [review]
Updated versions look good to me.

File: .changeset/fluffy-tips-cough.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@sap-ux/ui-components": patch
+"@sap-ux/prompting-ui": patch
```
Comment (devinea) [inline .changeset/fluffy-tips-cough.md:3]: axios-extension should also be released as a patch.

Comment (815are) [inline .changeset/fluffy-tips-cough.md:3]: there no public `@sap-ux/prompting-ui`  - it is private as examples ->
public is `@sap-ux/ui-prompting`

**Review by 815are — CHANGES_REQUESTED** [review]

File: .changeset/fluffy-tips-cough.md (hunk 1 above)
Comment (815are) [reply .changeset/fluffy-tips-cough.md:3]: I will create separate PR to rename examples package @sap-ux/prompting-ui to avoid mistakes

Comment (815are) [reply .changeset/fluffy-tips-cough.md:3]: looks like we are adding changelogs to packages in `examples` - https://github.com/SAP/open-ux-tools/blob/main/examples/odata-cli/CHANGELOG.md
Then my previous comment is not valid

Comment (hitesh-parmar) [reply .changeset/fluffy-tips-cough.md:3]: Thanks @devinea Yes correct as the linting auto-correct bot pushed a commit. I thought the linting bot only works on the `main` branch.

Comment (hitesh-parmar) [reply .changeset/fluffy-tips-cough.md:3]: @devinea I guess patch bump of axios is not required now as there is only linting change.

Comment (devinea) [reply .changeset/fluffy-tips-cough.md:3]: @hitesh-parmar I would release a change whenever there is a change to the `src` folder or dependencies.

Comment (hitesh-parmar) [reply .changeset/fluffy-tips-cough.md:3]: Done. Resolving this conversation.

**Review by devinea — APPROVED** [review]
Thanks for the updates. 
Changes lg.

**Review by nikmace — APPROVED** [review]
Changes look OK and are mostly test-related
Did not test manually

**Pull request merged**